assets/admin.css

```diff
@@ -0,0 +1,3 @@
+#post-body-content,
+#submitdiv .misc-pub-visibility,
+#submitdiv .misc-pub-curtime { display: none; }
```

Comment:
This is not according to coding standards, see https://engineering.hmn.md/standards/style/css/#basic-styles